extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -256,10 +257,19 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     const height = Number(this.element.getAttribute('height'));
     // Need to ensure these are numbers since width can be set to 'auto'.
     // Checking height just in case.
-    const useDefinedSizes = isExperimentOn(this.win, 'as-use-attr-for-format')
+    let useDefinedSizes = isExperimentOn(this.win, 'as-use-attr-for-format')
         && !this.isResponsive_()
         && !isNaN(width) && width > 0
         && !isNaN(height) && height > 0;
+    if (useDefinedSizes) {
+      if (Math.random() < 0.5) {
+        // In control: Do not use width / height attributes for size info.
+        useDefinedSizes = false;
```

Comment:
this reassignment isn't used anywhere that I see

Reply:
It's used on line 273, which is why I also can't use the predicate directly in 264.

Reply:
oh sorry, I see. I was looking just at the changes, my bad.